OmiseSwift/API Models/Capability.swift

```diff
@@ -99,8 +99,14 @@ extension Capability {
             case payNow
             case truemoney
             case payWithPointsCiti
+            case fpx
             case unknownSource(String, configurations: JSONDictionary)
         }
+        public struct BankDetails: Codable, Hashable {
+            public let code: String
+            public let name: String
+            public let active: Bool
```

Comment:
In swift it's common to prefix Boolean variables with `is`. So `active` would be named `isActive`.